src/data/modifiers.txt

```diff
@@ -8095,7 +8095,7 @@ Skill	Extra Brain	Mysticality Percent: +50
 Skill	Extra Innings	Adventures: +5
 # Extra Kidney: Booze gives 100% more Adventures
 Skill	Extra Muscles	Muscle Percent: +50
-# Extra Spleen: +5 spleen capacity
+Skill	Extra Spleen	Spleen Capacity: +5s
```

Comment:
```suggestion
Skill	Extra Spleen	Spleen Capacity: +5
```

Reply:
I should probably update data file consistency stuff to catch this...